lib/testmap.py

```diff
@@ -228,6 +228,8 @@ def contexts(image: str, *scenarios: Iterable[str], repo: str | None = None) ->
         ],
         '_manual': [
             'fedora-eln-boot',
+            *contexts('fedora-42-boot', ANACONDA_SCENARIOS),
```

Comment:
Did you actually test these?

Reply:
I am testing  here: https://github.com/rhinstaller/anaconda-webui/pull/631

Does not work yet. Therefore the manual trigger, when it passes in CI i will convert to automatic.

Reply:
oh yes. ok let me try to fix the tests today before merging.

Reply:
thanks!